clustermesh-apiserver/clustermesh/root.go

```diff
@@ -79,7 +80,7 @@ type parameters struct {
 	Resources      cmk8s.Resources
 	BackendPromise promise.Promise[kvstore.BackendOperations]
 	StoreFactory   store.Factory
-	SyncState      *SyncState
```

Comment:
Good call, I missed this while redoing this since it wasn't in the diffs :sweat_smile: 

I changed it from SyncState needing to be used as a pointer, to just making the internal SWG a pointer. I thought this would be a bit safer, guaranteeing that the SyncState will always use the same SWG whether pass-by-value or pointer. You could still create another SyncState with a different SWG by using `syncstate.Cell`